src/pages/search/advanced_search/ScientificValueFilter.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Checkbox, FormControlLabel } from '@mui/material';
+import React, { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { ResultParam } from '../../../api/searchApi';
+
+enum ScientificValueLevels {
+  LevelZero = 'Unassigned,LevelZero',
+  LevelOne = 'LevelOne',
+  LevelTwo = 'LevelTwo',
+}
+
+export const ScientificValueFilter = () => {
+  const [t] = useTranslation();
```

Comment:
Ikke at det gjør noe (med mindre react-i18n kommer med en breaking change), men greit for ¨å være konsistent med alle andre plasser
```suggestion
  const { t } = useTranslation();
```